hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java

```diff
@@ -504,20 +513,21 @@ public HddsProtos.DatanodeDetailsProto toProto(int clientVersion, Set<Port.Name>
    *                        If empty, all available ports will be included.
    * @return A {@link HddsProtos.DatanodeDetailsProto.Builder} Object.
    */
-
+  @VisibleForTesting
   public HddsProtos.DatanodeDetailsProto.Builder toProtoBuilder(
       int clientVersion, Set<Port.Name> filterPorts) {
```

Comment:
nit: I would like to omit `@VisibleForTesting`.  It tends to get outdated (it's easy to add a call in prod code and keep the annotation), and `toProtoBuilder` is safe to call for any code.